src/plugins/intel_gpu/src/kernel_selector/cl_kernels/dynamic_quantize_gpu_opt.cl

```diff
@@ -8,6 +8,14 @@
 #error "dynamic_quantize_gpu_opt.cl: Unsupported output dimension"
 #endif
 
+#define VLOAD_N CAT(vload, VEC_SIZE)
+#define VSTORE_N CAT(vstore, VEC_SIZE)
+#define HALF_N CAT(half, VEC_SIZE)
```

Comment:
what about using an existing macro such as `MAKE_VECTOR_TYPE(INPUT0_TYPE, VEC_SIZE)`?

Reply:
I will update it.

Reply:
Fixed to MAKE_VECTOR_TYPE(INPUT0_TYPE, VEC_SIZE)